delta/plugins/elasticsearch/src/main/resources/elasticsearch.conf

```diff
@@ -18,6 +18,11 @@ plugins.elasticsearch {
   priority = 2
   # the base uri to the Elasticsearch HTTP endpoint
   base = "http://localhost:9200"
+  # Basic authentication for Elasticsearch
+  #credentials {
+  #  username= "elastic_user"
+  #  credentials.password= "password"
```

Comment:
there's a duplicate credentials and the formatting seems a bit off (compared to the others)